### DISCUSSION TITLE
Project Name

### DISCUSSION THREAD

**Discussion by overheadhunter** [encryption-alliance/unified-vault-format#3] [General]
Since this should be a vendor-neutral project, we're looking for

* an organization name (something like "the secret society" - I actually like this one)
* a project/repo name (something like "awesome free encryption standard") which should also be the name of the final standard

Of course the repo should then be made public 😉

**Comment by dkocher** [comment]
I would propose `<project name>`**-alliance** for the organization name.

**Reply by overheadhunter** [reply]
Not very original, but tells what it is all about: `encrypted-file-alliance/encrypted-file-spec`

**Comment by dkocher** [comment]
What about `unified-file-encryption-allliance/unified-file-encryption-spec`.

**Reply by overheadhunter** [reply]
a little bulky in my opinion

**Reply by overheadhunter** [reply]
(at least for the organization name. repo name would work for me)

**Comment by overheadhunter** [comment]
How about `e2ee-alliance/(unified-)file-encryption-(spec|standard)`?

The orga name clearly states where the collaboration ends (anything that isn't encryption-related) and the repo name is concise and leaves room for additional future repos e.g. for `key-sharing-spec` that isn't directly related to the actual file format.

**Reply by dkocher** [reply]
Not sure about the acronym `e2ee`.

**Comment by rfjakob** [comment]
My take:

Portable Vault Format

**Reply by dkocher** [reply]
I like the `vault` part instead of `file-encryption`. Possible alterations `(portable|open)-vault-(format|spec|standard)`

**Reply by overheadhunter** [reply]
adding `unified`: `(portable|open|unified)-vault-(format|spec|standard)`

`unified-...-standard` seems redundant, though.

**Comment by overheadhunter** [comment]
my favourites:

`secret-council/open-vault-standard`
`secrecy-alliance/unified-vault-format`

Can you live with either of this?

**Comment by overheadhunter** [comment]
I'm moving this to `encryption-alliance/unified-vault-format`. It picks up most suggestions, tells exactly what this is all about and transports the message that it is a collaborative effort.
